pkg/operator/options/options.go

```diff
@@ -46,21 +46,23 @@ type FeatureGates struct {
 
 // Options contains all CLI flags / env vars for karpenter-core. It adheres to the options.Injectable interface.
 type Options struct {
-	ServiceName             string
-	MetricsPort             int
-	HealthProbePort         int
-	KubeClientQPS           int
-	KubeClientBurst         int
-	EnableProfiling         bool
-	DisableLeaderElection   bool
-	LeaderElectionNamespace string
-	MemoryLimit             int64
-	LogLevel                string
-	LogOutputPaths          string
-	LogErrorOutputPaths     string
-	BatchMaxDuration        time.Duration
-	BatchIdleDuration       time.Duration
-	FeatureGates            FeatureGates
+	ServiceName                    string
+	MetricsPort                    int
+	HealthProbePort                int
+	KubeClientQPS                  int
+	KubeClientBurst                int
+	EnableProfiling                bool
+	DisableLeaderElection          bool
+	LeaderElectionNamespace        string
+	MemoryLimit                    int64
+	LogLevel                       string
+	LogOutputPaths                 string
+	LogErrorOutputPaths            string
+	BatchMaxDuration               time.Duration
+	BatchIdleDuration              time.Duration
+	FeatureGates                   FeatureGates
+	MultinodeConsolidationTimeout  time.Duration
+	SinglenodeConsolidationTimeout time.Duration
```

Comment:
Could we rename these to `MultiNodeConsolidationTimeoutDuration` (capitalised `N` for `Node`) and `SingleNodeConsolidationTimeout`? This would be similar to the previous const and keep consistency with the `MultiNodeConsolidation` struct.